libs/blocks/article-header/article-header.js

```diff
@@ -126,6 +126,8 @@ async function buildSharing() {
 async function validateDate(date) {
   if (date && !/^[0-1]\d{1}-[0-3]\d{1}-[2]\d{3}$/.test(date.textContent.trim())) {
     // match publication date to MM-DD-YYYY format
+    const { env } = getConfig();
+    if (env?.name === 'prod') return;
     date.classList.add('article-date-invalid');
     date.setAttribute('title', await replaceKey('invalid-date', getConfig()));
   }
```

Comment:
Let's try to avoid nested IFs
```suggestion
async function validateDate(date) {
  const { env } = getConfig();
  if (!date || /^[0-1]\d{1}-[0-3]\d{1}-[2]\d{3}$/.test(date.textContent.trim()) || env?.name === "prod) return
  // match publication date to MM-DD-YYYY format
  date.classList.add('article-date-invalid');
  date.setAttribute('title', await replaceKey('invalid-date', getConfig()));
  }
```

Reply:
Good call, thanks